From 6cd538fd226819761927bfec0003fbd3f6d03cc2 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 07 Apr 2019 13:08:41 +0300
Subject: [PATCH] [SW-0] goya: pll: fix white space errors

Several checkpatch errors are now spotted by jenkins each time we
change the goya/pll.c file.
Fix them, so we have clean builds.

Change-Id: Ic2fba6b41cc843743c45d915aea7716a8c361213
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya/pll.c b/goya/pll.c
index df613e0..0449ea9 100644
--- a/goya/pll.c
+++ b/goya/pll.c
@@ -57,12 +57,19 @@
 };
 
 static struct hl_pll_cfg goya_pll_cfg[] = {
-	{.name = "CPU", .base = (void *)mmCPU_PLL_BASE,
-	 .nr = 0, .nf = 47, .od = 1, .nb = 23, .div_factor = {0, -1, 1, -1} },
-	{.name = "IC", .base = (void *)mmIC_PLL_BASE,
-	 .nr = 0, .nf = 63, .od = 15, .nb = 31, .div_factor = {0, 1, 1, 2} },
-	{.name = "MME", .base = (void *)mmPSOC_MME_PLL_BASE,
-	 .nr = 0, .nf = 63, .od = 15, .nb = 31, .div_factor = {0, 1, 2, 8} },
+	{
+		.name = "CPU", .base = (void *)mmCPU_PLL_BASE,
+		.nr = 0, .nf = 47, .od = 1, .nb = 23,
+		.div_factor = { 0, -1, 1, -1 }
+	}, {
+		.name = "IC", .base = (void *)mmIC_PLL_BASE,
+		.nr = 0, .nf = 63, .od = 15, .nb = 31,
+		.div_factor = { 0, 1, 1, 2 }
+	}, {
+		.name = "MME", .base = (void *)mmPSOC_MME_PLL_BASE,
+		.nr = 0, .nf = 63, .od = 15, .nb = 31,
+		.div_factor = { 0, 1, 2, 8 }
+	},
 };
 
 static int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
